src/renderer/src/components/Map.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useRef } from 'react'
+
+import 'maplibre-gl/dist/maplibre-gl.css'
+
+import maplibregl from 'maplibre-gl'
+
+import { useSharedLocationContext } from '../contexts/SharedLocationContext'
+
+export const Map = () => {
+	const mapContainerRef = useRef<HTMLDivElement | null>(null)
+	const mapRef = useRef<maplibregl.Map | null>(null)
+	const { location, isLoading } = useSharedLocationContext()
+
+	useEffect(() => {
+		if (!mapContainerRef.current) return
+		const map = new maplibregl.Map({
+			container: mapContainerRef.current,
+			style: 'https://demotiles.maplibre.org/style.json',
```

Comment:
ideally the style that's used comes from `useMapStyleUrl()` in `@comapeo/core-react`. By default, that provides a Mapbox-based map as the default, which I _think_ should work with MapLibreGL but I'm not actually super sure.